src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessorTest.java

```diff
@@ -26,7 +25,6 @@
 import static uk.gov.hmcts.reform.blobrouter.data.model.Status.DISPATCHED;
 import static uk.gov.hmcts.reform.blobrouter.testutils.DirectoryZipper.zipAndSignDir;
 
-@ActiveProfiles("db-test")
```

Comment:
this test does use database

Reply:
It does not need DbHelper - the only component linked to this profile.

Reply:
perhaps this profile can be removed then (not only profile usage)

Reply:
This profile makes sense, it is limiting instantiation of DbHelper.

Reply:
It would be best if this test did DB cleanup at the start. It's not strictly related with this PR, but it needs to be fixed. I think we need that profile reference here in the end, so that we can use DB Helper.